XenAdmin/TabPages/ManageUpdatesPage.cs

```diff
@@ -502,8 +516,7 @@ into g
         private void DeleteAllAllertAction_Completed(ActionBase sender)
         {
             Program.Invoke(Program.MainWindow, () =>
-            {                
-                Rebuild();

```

Comment:
Thank you for the fix @kc284 . I think these changes are good and dismiss updates now works as intended with the columns resizing as necessary. However there is a regression introduced by removing this Rebuild call. It is needed to set the 'No Updates Found' message/warning on the grid after all the alerts are dismissed. Maybe a full rebuild is not necessary, but the bit of code that sets the icon and message are needed, because at the moment it just shows an empty grid with no message.